plugins/net_plugin/net_plugin.cpp

```diff
@@ -4192,10 +4247,20 @@ namespace eosio {
 
                std::string extra_listening_log_info =
                      ", max clients is " + std::to_string(my->connections.get_max_client_count());
-                     
+
+               auto listen_addr = address;
+               auto limit = string("0");
+               if( std::count(address.begin(), address.end(), ':') > 1 ) {
```

Comment:
This doesn't work for IPv6, right? As IPv6 addresses are of the form: `2001:db8:3333:4444:5555:6666:7777:8888`

Reply:
Now requiring IPV6 square bracket format. (The trailing one, anyway.)

Reply:
Added IPv6 example.  Sample address is taken from the IPv6 address Wikipedia page.